tests/python/pants_test/backend/python/tasks2/test_pytest_run_integration.py

```diff
@@ -28,52 +28,51 @@ def test_pytest_run_conftest_succeeds(self):
                                 'testprojects/tests/python/pants/conf_test'])
     self.assert_success(pants_run)
 
-  @unittest.skip('https://github.com/pantsbuild/pants/issues/3255')
   def test_pytest_run_timeout_fails(self):
     start = time.time()
     pants_run = self.run_pants(['clean-all',
                                 'test.pytest',
                                 '--test-pytest-coverage=auto',
                                 '--test-pytest-options=-k exceeds_timeout',
-                                '--timeout-default=1',
+                                '--test-pytest-timeout-default=1',
+                                '--cache-test-pytest-ignore',
                                 'testprojects/tests/python/pants/timeout:exceeds_timeout'])
     end = time.time()
     self.assert_failure(pants_run)
 
-    # Ensure that the failure took less than 20 seconds to run.
-    self.assertLess(end - start, 20)
+    # Ensure that the failure took less than 35 seconds to run to allow for test overhead.
```

Comment:
If possible, going really excessive here to minimize flakiness would be good... ie, have the test sleep for 1000 seconds, and ensure that it takes fewer than 200.

Reply:
done.